### DISCUSSION TITLE
execinquery repo has been archived

### DISCUSSION THREAD

**Discussion by sahidvelji** [golangci/golangci-lint#4575] [General] [answered]
https://github.com/1uf3/execinquery
Just noticed that this repo has been archived, so I think this linter can be marked deprecated in the next version?

**Comment by ccoVeille** [comment]
Or forked or imported. I'm unsure if the team has information about the usage of each linter

**Comment by ldez** [comment]
I will analyze the situation.

**Reply by ldez** [reply, marked as answer]
After a private discussion with the author of the linter, we will deprecate this linter.
https://github.com/golangci/golangci-lint/pull/4652
